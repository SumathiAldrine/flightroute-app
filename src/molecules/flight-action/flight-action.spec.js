import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import {configure } from 'enzyme';
import renderer from 'react-test-renderer';
import FlightAction from './index.js';
configure({adapter: new Adapter()});

describe('Test FlightAction Component', () => {
  it('Test click event', () => {
    const mockCallBack = jest.fn();
    const tree = renderer.create(<FlightAction
      flightNo="1"
      index={0}
      selected={false}
      handleSelect={mockCallBack}
      isMobile= {false}
  />).toJSON();
  
    expect(tree).toMatchSnapshot();
  });

  it('Test click event for Mobile', () => {
    const mockCallBack = jest.fn();
    const tree = renderer.create(<FlightAction
      flightNo="1"
      index={0}
      selected={false}
      handleSelect={mockCallBack}
      isMobile= {true}
  />).toJSON();
  
    expect(tree).toMatchSnapshot();
  });
});