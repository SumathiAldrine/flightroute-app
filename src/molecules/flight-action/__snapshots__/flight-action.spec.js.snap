// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test FlightAction Component Test click event 1`] = `
<div
  className="flight-action__select"
>
  <div
    className="flightNo"
    tabIndex="0"
  >
    1
  </div>
  <button
    className="cta"
    onClick={[MockFunction]}
  >
    Select
  </button>
</div>
`;

exports[`Test FlightAction Component Test click event for Mobile 1`] = `
<div
  className="flight-action__select-mobile"
>
  <div
    className="flightNo-mobile"
    tabIndex="0"
  >
    1
  </div>
  <button
    className="cta"
    onClick={[MockFunction]}
  >
    Select
  </button>
</div>
`;
